source/extensions/filters/http/transformation/transformation_filter_config.cc

```diff
@@ -29,7 +29,7 @@ TransformerConstSharedPtr Transformation::getTransformer(
   case envoy::api::v2::filter::http::Transformation::kTransformerConfig: {
     auto &factory = Config::Utility::getAndCheckFactory<TransformerExtensionFactory>(transformation.transformer_config());
     auto config = Config::Utility::translateAnyToFactoryConfig(transformation.transformer_config().typed_config(), context.messageValidationContext().staticValidationVisitor(), factory);
-    return factory.createTransformer(*config, context);
+    return factory.createTransformer(*config, transformation.log_request_response_info(), context);
```

Comment:
This line is called when we interact with a transformer defined outside of envoy-gloo. Without this change, that transformer will be created without knowledge of whether `log_request_response_info` has been set.

Reply:
This is really the most important part of the PR -- the rest is just changing the signature of `createTransformer` and  updating the places where it's called to match that change